frontend/src/components/Home/ApartmentCard.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { ReactElement } from 'react';
+import { Card, CardImg } from 'react-bootstrap';
+import ApartmentImg from '../../assets/apartment-sample.png';
+import { ReactComponent as MapIcon } from '../../assets/map-pin.svg';
+import { ReactComponent as PriceIcon } from '../../assets/tag-outline.svg';
+import { Container, Row, Col } from 'react-bootstrap';
+type Props = {
+  address: string;
+  company?: string;
+  bedsAndBaths?: string;
+  price?: string;
+  topReviewDisplay?: string;
+  numReviews?: string;
+};
+
+const ApartmentCard = (props: Props): ReactElement => {
+  function renderAdressSection() {
+    return (
+      <Row>
+        <Col xs={8}>
+          <Row>
+            <MapIcon className="pr-1 mt-1" />
+            <Card.Title className="pb-1 mt-0 mb-1" style={{ fontSize: '21px' }}>
+              {props.address}
+            </Card.Title>
+          </Row>
+        </Col>
+        {renderPrice()}
+      </Row>
+    );
+  }
+
+  function renderPrice() {
+    return props.price ? (
+      <Col style={{ paddingRight: '0px', paddingLeft: '10px' }} xs={4}>
+        <Row>
+          <PriceIcon style={{ marginLeft: '12px', marginBottom: '5px' }} />
+          <Card.Subtitle
+            className="ml-1"
+            style={{
```

Comment:
Try not to use inline styles but put them in a separate CSS file i.e. `ApartmentCard.module.css`.